pages/common/clockwork-cli.md

```diff
@@ -0,0 +1,15 @@
+# clockwork-cli
+
+> A command line interface for the Clockwork PHP debugging framework.
+
+- Monitor Clockwork logs for the current project:
+
+`clockwork-cli`
+
+- Monitor Clockwork logs for a specific project:
+
+`clockwork-cli {{path/to/directory}}`
+
+- Monitor Clockwork logs for multiple projects:
+
+`clockwork-cli {{path/to/directory_1}} {{path/to/directory_2}}`
```

Comment:
```suggestion
`clockwork-cli {{path/to/directory1 path/to/directory2 …}}`
```

As per #2745, I would suggest this change. Only a consistency suggestion though, the example looks clear to me either way.

Reply:
I've just noticed one thing with using the unicode character, it doesn't seem to render as underlined text on the node client.
![image](https://user-images.githubusercontent.com/1899334/52198153-d5762180-2859-11e9-8552-ef628574aea6.png)
![image](https://user-images.githubusercontent.com/1899334/52198170-dc049900-2859-11e9-8c64-7faaa99c4d15.png)

Reply:
@pxgamer oh, damn... that could be a limitation of ASCII escape codes? Perhaps we could use the triple dot to improve compatibility or make the client translate it. IDK. That's a separate issue though, should be proposed as an issue and talked about.

Reply:
Hrm. That's a bug in the client then.